### PR TITLE
[iOS] [Issue 125] Add list items

### PR DESCRIPTION
**Pull request by cyndichin**
## Description
**Implements issue #125** 

Created `ActivityView` to hold the mock data to populate the list items and created `ActivityItemView` to display the activity name and duration for each item in the list.

## Screenshots
| Light Mode | Dark Mode | 
| --- | --- |
| simulator_screenshot_607E1356-C32C-4E06-B548-52632BB9E189 | simulator_screenshot_9B25A277-5284-46DA-88E2-F9028401409C |

### CODE REVIEW

**Comment by cyndichin** [comment]
@clc80 @devanshimodha this PR is ready for review at your convenience. thank you!

File: coding-projects/ios/TaskTracker/TaskTracker/List Screen/ListView.swift [hunk 1]
```diff
@@ -8,8 +8,56 @@
...
+            // TODO: Display list items in sections based on date #126
+
+            ActivityItemView(name: activity.name, duration: activity.duration)
+                .listRowSeparator(.hidden)
+        }
+        .listStyle(.plain)
+    }
+}
+
+struct Activity: Identifiable {
```
Comment (clc80) [inline coding-projects/ios/TaskTracker/TaskTracker/List Screen/ListView.swift:40]: This Activity struct would make more sense in the ListModel.swift file. The view shouldn't really be responsible for holding models.

**Review by clc80 — CHANGES_REQUESTED** [review]
This is a great start, I do have one change in regards to moving some code around.

File: coding-projects/ios/TaskTracker/TaskTracker/List Screen/ListView.swift [hunk 2]
```diff
@@ -8,8 +8,56 @@
 import SwiftUI
 
 struct ListView: View {
+    let activities: [Activity] = [
```
Comment (clc80) [inline coding-projects/ios/TaskTracker/TaskTracker/List Screen/ListView.swift:11]: This should probably go in the model file with the struct.

**Review by devanshimodha — APPROVED** [review]

**Pull request merged**